packages/cardano-services-client/test/DRepProvider/util.ts

```diff
@@ -0,0 +1,12 @@
+export const mockResponses = (request: jest.Mock, responses: [string | RegExp, unknown][]) => {
```

Comment:
Please hoist https://github.com/input-output-hk/cardano-js-sdk/blob/7198c54ba330233b553501a7d09e0d7ccc4f27ea/packages/cardano-services-client/test/AssetInfoProvider/util.ts#L4 and use that for this provider test. I'll rebase https://github.com/input-output-hk/cardano-js-sdk/pull/1532 to use that change too

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1534/commits/574cf3c6ed28010cfa65e10c8391f99340744ac4